.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: run-tests
+on: [pull_request]
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+          python-version: [ 3.6, 3.7, 3.8 ]
+          include:
+            - os: ubuntu-latest
+              set_display: export DISPLAY=:99; Xvfb :99 -screen 0 1024x768x24 -ac -noreset & sleep 3
```

Comment:
Lines 9 through 11 can be removed since you are not running any tests that require a screen